TODO.md

```diff
@@ -0,0 +1,92 @@
+## TODO:
```

Comment:
This file was removed in the target branch, so only "TO QA" section should remain